R/test-files.r

```diff
@@ -40,10 +40,11 @@ test_files <- function(paths, reporter = "summary",
   if (length(paths) == 0) stop('No matching test file in dir')
 
   current_reporter <- find_reporter(reporter)
-  current_reporter$start_reporter()
-  results <- lapply(paths, test_file, env = env,
-    reporter = current_reporter, start_end_reporter = FALSE)
-  current_reporter$end_reporter()
+  with_reporter(
```

Comment:
I think this would be clearer as `results <- with_reporter(current_reporter, ...)`